tests/test_common.py

```diff
@@ -4,8 +4,33 @@
 """
 t3 tests test_utils module
 """
+import os
+from typing import Optional
+
+from arc.common import read_yaml_file
 
 import t3.common as common
+from t3.common import DATA_BASE_PATH, EXAMPLES_BASE_PATH
+from t3.main import T3
+
+
+def run_minimal(project: Optional[str] = None,
```

Comment:
Should this function be imported from test_main.py? Or should it be relocated to a common file?

Reply:
Yea that's smart. I'll move `run_minimal` to `test_main`